addons/main/weapons/CfgWeapAR15.hpp

```diff
@@ -135,4 +150,10 @@ class hlc_wp_XM177E2: hlc_M16Base {
     class WeaponSlotsInfo: WeaponSlotsInfo {
         mass = 65;
     };
+    class FullAuto: Burst {
+      dispersion =  0.0022;
```

Comment:
Single spaces please.
```suggestion
      dispersion = 0.0022;
```